orm_lib/inc/drogon/orm/Exception.h

```diff
@@ -137,15 +137,24 @@ class SqlError : public Failure
     /// SQLSTATE string describing the error type, if known; or empty string.
     const std::string sqlState_;
 
+    const int errcode_;
+    const int extended_errcode_;
+
   public:
     DROGON_EXPORT explicit SqlError(const std::string &msg = "",
                                     const std::string &Q = "",
                                     const char sqlstate[] = nullptr);
+    DROGON_EXPORT explicit SqlError(const std::string &msg,
+                                    const std::string &Q,
+                                    const int errcode,
+                                    const int extended_errcode);
     DROGON_EXPORT virtual ~SqlError() noexcept;
 
     /// The query whose execution triggered the exception
     DROGON_EXPORT const std::string &query() const noexcept;
     DROGON_EXPORT const std::string &sqlState() const noexcept;
+    DROGON_EXPORT const int &errcode() const noexcept;
+    DROGON_EXPORT const int &extended_errcode() const noexcept;
```

Comment:
Please use lowerCamelCase.

Reply:
Please return the value directly; for the int type, returning a reference offers no performance benefit.